src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -32,11 +35,40 @@ namespace Utils
             rocksdb::Options options;
             options.create_if_missing = true;
             rocksdb::DB* dbRawPtr;
+            std::vector<rocksdb::ColumnFamilyDescriptor> columnsDescriptors;
+            const std::filesystem::path databasePath {dbPath};
 
             // Create directories recursively if they do not exist
-            std::filesystem::create_directories(std::filesystem::path(dbPath));
+            std::filesystem::create_directories(databasePath);
 
-            const auto status {rocksdb::DB::Open(options, dbPath, &dbRawPtr)};
+            // Get a list of the existing columns descriptors.
+            const auto DB_FILE {databasePath / "CURRENT"};
```

Comment:
From the codding style:
> Constants must use ALL_CAPS_WITH_UNDERSCORES (defines or constexpr).

I think it should be camelCase

Reply:
Changed!